.mbedignore

```diff
@@ -0,0 +1,2 @@
+test/*
```

Comment:
where this comes from `tests/` ?

Reply:
All these comments are from one git subtree try.. I will fix these